lib/tasks/cms/cms.rake

```diff
@@ -210,6 +210,13 @@ namespace :cms do
         puts "\t++ Fixing pagination href in #{page.id}/shared/pagination"
       end
     end
+
+    desc 'Fixes invalid system names in sections'
+    task :section_invalid_system_names => :environment do
+      CMS::Section.unscoped.find_each do |section|
+        section.update_column(:system_name, "section-#{section.id}") unless section.valid?
```

Comment:
By any chance, is there an easy way to figure out which sections are invalid with a DB Query so we don't have to go over all the records?
If not, I think the 1000 records batch size should be safe enough :v:

Reply:
If memory serves right, we can do something like `where("column ~* ?", 'regex')` BUT I'm not sure it works on MySQL (🤦) and having DB specific code should be avoided.
So this is cool for me!

Reply:
If memory serves right, we can do something like `where("column ~* ?", 'regex')` BUT I'm not sure it works on MySQL (🤦) and having DB specific code should be avoided.
So this is cool for me!